### PR TITLE
[eBus] Fix expanded bytes

### PR DESCRIPTION
**Pull request by csowada**

### CODE REVIEW

**Comment by watou** [comment]
The convert CR LF to LF commit might be the one causing conflicts.  Please rebase this PR to latest master, and please write up in detail what this problem is with existing ebus binding and what your fix does to correct it.  Thank you!

**Comment by csowada** [comment]
Okay, rebased on the latest version.

File: bundles/binding/org.openhab.binding.ebus/src/main/java/org/openhab/binding/ebus/internal/connection/EBusCommandProcessor.java [hunk 1]
```diff
@@ -160,7 +160,7 @@ public void setConfigurationProvider(EBusConfigurationProvider configurationProv
      * @param values
      * @return
      */
-    private byte[] composeEBusTelegram(String commandId, Byte dst, Byte src, Map<String, Object> values) {
+    public byte[] composeEBusTelegram(String commandId, Byte dst, Byte src, Map<String, Object> values) {
```
Comment (watou) [inline bundles/binding/org.openhab.binding.ebus/src/main/java/org/openhab/binding/ebus/internal/connection/EBusCommandProcessor.java]: Not a critical point, but why increase the method visibility?

File: bundles/binding/org.openhab.binding.ebus/src/main/java/org/openhab/binding/ebus/internal/utils/EBusUtils.java [hunk 2]
```diff
@@ -249,6 +249,11 @@ static public EBusTelegram processEBusData(byte[] data) {
             int crcPos = nnPos + 1;
             byte crc = data[crcPos];
 
+            if (uc_crc == (byte) 0xAA && crc == (byte) 0xA9 && data[++crcPos] == (byte) 0x01) {
+                // replace with original value
+                crc = (byte) 0XAA;
```
Comment (watou) [inline bundles/binding/org.openhab.binding.ebus/src/main/java/org/openhab/binding/ebus/internal/utils/EBusUtils.java]: If you do another commit for this PR, please change to `0xAA` for consistency.

File: bundles/binding/org.openhab.binding.ebus/src/main/java/org/openhab/binding/ebus/internal/utils/EBusUtils.java [hunk 3]
```diff
@@ -321,6 +326,11 @@ static public EBusTelegram processEBusData(byte[] data) {
             crc = data[data.length - 3];
             buffer.put(data, data.length - 3, 3);
 
+            if (uc_crc == (byte) 0xAA && crc == (byte) 0xA9 && data[++crcPos] == (byte) 0x01) {
+                // replace with original value
+                crc = (byte) 0XAA;
```
Comment (watou) [inline bundles/binding/org.openhab.binding.ebus/src/main/java/org/openhab/binding/ebus/internal/utils/EBusUtils.java]: If you do another commit for this PR, please change to `0xAA` for consistency.

**Comment by csowada** [comment]
So, I changed the two files and I have rebased the PR. I use this version since weeks in my productive environment. But I can ask in the community for feedback.

**Comment by watou** [comment]
Thank you @csowada.  If there is any doubt in your mind that this might cause a regression for other users, please see if others can test it by posing at https://community.openhab.org.  Otherwise, this PR LGTM.

**Comment by watou** [comment]
cherry picked onto 1.9.x
